src/coreclr/nativeaot/BuildIntegration/Microsoft.NETCore.Native.targets

```diff
@@ -80,8 +73,8 @@ The .NET Foundation licenses this file to you under the MIT license.
     <FrameworkLibPath Condition="'$(FrameworkLibPath)' == ''">$(NativeOutputPath)</FrameworkLibPath>
     <FrameworkObjPath Condition="'$(FrameworkObjPath)' == ''">$(NativeIntermediateOutputPath)</FrameworkObjPath>
 
-    <SharedLibrary Condition="'$(TargetOS)' == 'windows'">$(FrameworkLibPath)\Framework$(LibFileExt)</SharedLibrary>
-    <SharedLibrary Condition="'$(TargetOS)' != 'windows'">$(FrameworkLibPath)\libframework$(LibFileExt)</SharedLibrary>
+    <SharedLibrary Condition="'$(TargetOS)' == 'windows'">$(FrameworkLibPath)\Framework$(StaticLibSuffix)</SharedLibrary>
+    <SharedLibrary Condition="'$(TargetOS)' != 'windows'">$(FrameworkLibPath)\libframework$(StaticLibSuffix)</SharedLibrary>
```

Comment:
@jkotas, the name of this property has `shared` but the actual value of file extension was (and is) was static: ('.lib' for windows and '.a' for unix). Is this a typo in the property name?

Reply:
I do not think it is a typo. "shared" here means "shared between multiple projects".